.travis.yml

```diff
@@ -32,14 +32,17 @@ script:
 after_success:
     - bash <(curl -s https://codecov.io/bash)
 
-before_deploy: make dist -j 4
+before_deploy: make dist -j 4 && cp build/psysh/psysh dist/psysh.phar && cp build/psysh/psysh.asc dist/psysh.asc
```

Comment:
As files already exist in `build/psysh/psysh.asc` and `build/psysh/psysh.asc`, this (copying) is actually not needed, as I can directly release them to GitHub Releases.

Reply:
This is now fixed.

Reply:
Why does it need to be named psysh.phar? I’d really rather not.

Reply:
Because PHIVE requires that.

Why you don't want this? In case of installation with PHIVE, `.phar` will be dropped so user when saving wouldn't notice it.

Reply:
For everyone who looks at the github release and _doesn’t_ use phive. It means the most downloadable looking thing in the release is a `.phar`, which is bound to cause confusion. I’ve never loved this about phive. I feel like it forces suboptimal experiences for non-phive users, unless I want to maintain my own repo server.

Reply:
But is this actually a problem? Because your website already shows installation with `wget` and `composer`. And if users need compat versions, they will probably already know that.

However, if you don't want that, you can set up [custom repository server](https://phar.io/distribute-your-own.html). I think that you can just use GitHub Pages for this with some CI script that automatically updates them at the new release.  If you think this would be better, I can try to create new PR for it.